==14105== Memcheck, a memory error detector
==14105== Copyright (C) 2002-2011, and GNU GPL'd, by Julian Seward et al.
==14105== Using Valgrind-3.7.0 and LibVEX; rerun with -h for copyright info
==14105== Command: claws-mail
==14105== Parent PID: 32435
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==14105==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==14105==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==14105==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==14105==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==14105==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==14105==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==14105==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==14105==    by 0x4D8CDA: main (main.c:1265)
==14105==  Address 0x1192c8f8 is 0 bytes after a block of size 40 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==14105==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==14105==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==14105==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==14105==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==14105==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==14105==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==14105==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==14105==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==14105==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==14105==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==14105==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==14105==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==14105==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==14105==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==14105==    by 0x4D8CDA: main (main.c:1265)
==14105==  Address 0x1192cc78 is 0 bytes after a block of size 40 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x66038F: mgu_list_test_unq_nc (mgutils.c:383)
==14105==    by 0x63D33A: ldapctl_criteria_list_add (ldapctrl.c:344)
==14105==    by 0x611855: addrindex_parse_ldap_attrlist (addrindex.c:1331)
==14105==    by 0x611CD2: addrindex_parse_ldap (addrindex.c:1443)
==14105==    by 0x612371: addrindex_read_index (addrindex.c:1612)
==14105==    by 0x6126A6: addrindex_read_file (addrindex.c:1684)
==14105==    by 0x613DB5: addrindex_read_data (addrindex.c:2307)
==14105==    by 0x609995: addressbook_read_file (addressbook.c:4170)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA27: wcslen (wcslen.S:55)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x135a8cf8 is 0 bytes after a block of size 56 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA27: wcslen (wcslen.S:55)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x135a8d78 is 0 bytes after a block of size 56 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x135af0f8 is 0 bytes after a block of size 40 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x135af7e8 is 40 bytes inside a block of size 44 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5DCFC2: toolbar_ret_val_from_text (toolbar.c:276)
==14105==    by 0x5DD50E: toolbar_parse_item (toolbar.c:372)
==14105==    by 0x5DE331: toolbar_read_config_file (toolbar.c:750)
==14105==    by 0x5E1ABD: toolbar_create (toolbar.c:1915)
==14105==    by 0x4E58AA: main_window_create (mainwindow.c:1899)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==14105==    by 0x5B8A78: summary_init (summaryview.c:997)
==14105==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x152e8bb0 is 32 bytes inside a block of size 36 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==14105==    by 0x5B8A78: summary_init (summaryview.c:997)
==14105==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==14105==    by 0x5B8A78: summary_init (summaryview.c:997)
==14105==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x152e8ce0 is 32 bytes inside a block of size 36 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB0C1: summary_tag_cmp_list (summaryview.c:6190)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x5CB104: summary_tags_menu_create (summaryview.c:6204)
==14105==    by 0x5B8A78: summary_init (summaryview.c:997)
==14105==    by 0x4E6B00: main_window_create (mainwindow.c:2180)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==14105==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x15371dd0 is 32 bytes inside a block of size 36 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80AE3EB: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==14105==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==14105==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105==  Address 0x15371f00 is 32 bytes inside a block of size 36 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFC3: mainwin_tag_cmp_list (mainwindow.c:1246)
==14105==    by 0x80AE419: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4DEFFE: mainwindow_tags_menu_create (mainwindow.c:1258)
==14105==    by 0x4E6C04: main_window_create (mainwindow.c:2202)
==14105==    by 0x4D8D20: main (main.c:1286)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA27: wcslen (wcslen.S:55)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==  Address 0x15820090 is 48 bytes inside a block of size 52 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==  Address 0x158207a0 is 32 bytes inside a block of size 36 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA0F: wcslen (wcslen.S:48)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E01DA: sink (gtksctree.c:1082)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==  Address 0x158208c0 is 32 bytes inside a block of size 36 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E01DA: sink (gtksctree.c:1082)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA27: wcslen (wcslen.S:55)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==14105==  Address 0x15824748 is 0 bytes after a block of size 56 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6AD12F: gtk_cmctree_post_recursive (gtkcmctree.c:3330)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA3F: wcslen (wcslen.S:62)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E01DA: sink (gtksctree.c:1082)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==14105==    by 0x4B2AC2: folderview_set_folders (folderview.c:975)
==14105==  Address 0x15829b88 is 0 bytes after a block of size 72 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E01DA: sink (gtksctree.c:1082)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA3F: wcslen (wcslen.S:62)
==14105==    by 0x89B4DA8: wcscoll_l (strcoll_l.c:510)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E0320: heap_sort (gtksctree.c:1109)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==14105==    by 0x4B2AC2: folderview_set_folders (folderview.c:975)
==14105==  Address 0x1582b5b8 is 0 bytes after a block of size 72 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE6B: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E0180: sink (gtksctree.c:1078)
==14105==    by 0x6E0320: heap_sort (gtksctree.c:1109)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==14105== 
==14105== Invalid read of size 8
==14105==    at 0x89AAA6B: wcslen (wcslen.S:75)
==14105==    by 0x89B4D99: wcscoll_l (strcoll_l.c:509)
==14105==    by 0x80BDE79: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E01DA: sink (gtksctree.c:1082)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==14105==    by 0x4B2AC2: folderview_set_folders (folderview.c:975)
==14105==  Address 0x1698a518 is 0 bytes after a block of size 104 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x80986E0: g_malloc (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BEB20: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x80BDE54: g_utf8_collate (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x4B7F04: folderview_clist_compare (folderview.c:2498)
==14105==    by 0x6E01DA: sink (gtksctree.c:1082)
==14105==    by 0x6E02A2: heap_sort (gtksctree.c:1103)
==14105==    by 0x6E0462: stree_sort (gtksctree.c:1145)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6E07C7: gtk_sctree_sort_recursive (gtksctree.c:1205)
==14105==    by 0x4B4F4B: folderview_sort_folders (folderview.c:1715)
==14105==    by 0x4B50FD: folderview_append_folder (folderview.c:1739)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cc9b020 is 80 bytes inside a block of size 2,731 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cc9b020 is 80 bytes inside a block of size 2,731 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E64A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cc9c020 is 16 bytes inside a block of size 200 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67AEBB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A750: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B257: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1ccd4020 is 16 bytes before a block of size 280 alloc'd
==14105==    at 0x4C2B3F8: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6C13D8: _PyObject_GC_Malloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C74CC: _PyObject_GC_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6651F3: PyDict_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C637FCC: PyModule_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6380F1: PyImport_AddModule (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBB05: PyImport_ExecCodeModuleEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B77: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1ccf4020 is 672 bytes inside a block of size 768 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6C1EB9: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5806D7: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBB7C: PyImport_ExecCodeModuleEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B77: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cda7020 is 9,360 bytes inside a block of size 9,383 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cda7020 is 9,360 bytes inside a block of size 9,383 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E64A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cda7020 is 9,360 bytes inside a block of size 9,383 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5EA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cc60020 is 64 bytes inside a block of size 1,864 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6A3544: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B273: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1ce3b020 is 8 bytes after a block of size 280 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67AEBB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A73C: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B257: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6958EB: PyGrammar_AddAccelerators (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695BD7: PyParser_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6802D4: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6958EB: PyGrammar_AddAccelerators (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695BD7: PyParser_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6802D4: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6958EB: PyGrammar_AddAccelerators (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695BD7: PyParser_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6802D4: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1ceeb020 is 384 bytes inside a block of size 676 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6958EB: PyGrammar_AddAccelerators (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695BD7: PyParser_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6802D4: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf4b020 is 16 bytes inside a block of size 384 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695FD7: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf58020 is 35,456 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695FD7: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf58020 is 35,456 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DEB40: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CA9A7: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C598328: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf58020 is 35,456 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DEB40: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CBA5B: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5983AB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf58020 is 35,456 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf59020 is 144 bytes inside a block of size 768 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DEB40: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CBA5B: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5983AB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf58020 is 35,456 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C69540B: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620032: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C69541F: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620032: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf80020 is 34,288 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695F14: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf80020 is 34,288 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DE8A6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CB707: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CA973: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CBA5B: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CBB29: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CA9A7: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C598328: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf82020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A3466: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584771: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBB7C: PyImport_ExecCodeModuleEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cf74020 is 4,816 bytes inside a block of size 8,192 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C698999: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C698A92: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6ACCCC: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57EDE5: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6495E6: _PyString_Resize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E5804: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FD5: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cfda020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E5E42: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FD5: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C580537: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1cfda020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C681F75: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C639874: _PyCodec_Lookup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C639A08: PyCodec_Encoder (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C590247: Py_InitializeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B0F6: plugin_init (python_plugin.c:661)
==14105==  Address 0x1d023020 is 48 bytes inside a block of size 280 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C607ADD: PyList_Append (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67AEBB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A750: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A194: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67A73C: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B257: PyMarshal_ReadObjectFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C67B35D: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EC9: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d083020 is 304 bytes inside a block of size 320 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620220: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d083020 is 304 bytes inside a block of size 320 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620220: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695FD7: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d083020 is 304 bytes inside a block of size 320 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620220: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695F14: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d083020 is 304 bytes inside a block of size 320 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620220: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69601D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==  Address 0x1d083020 is 304 bytes inside a block of size 320 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620220: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C696035: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d083020 is 304 bytes inside a block of size 320 free'd
==14105==    at 0x4C2B4F0: realloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C695613: PyNode_AddChild (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C620220: PyParser_AddToken (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680351: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695F79: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695F79: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695FA8: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d082020 is 34,608 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0cd020 is 96 bytes inside a block of size 384 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0cd020 is 96 bytes inside a block of size 384 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CB5AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5CA9A7: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5975F1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A60F: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C598F97: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==  Address 0x1d0cd020 is 96 bytes inside a block of size 384 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C652DC5: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599C4C: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0d0020 is 96 bytes inside a block of size 384 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6BF74D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5995B3: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599BAF: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0d2020 is 624 bytes inside a block of size 768 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6A369A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6BF896: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C652DC5: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599C4C: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32932C: cmpy_add_node (nodetype.c:88)
==14105==    by 0x1C3263CF: initclawsmail (clawsmailmodule.c:926)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C696035: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==14105==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0e1020 is 34,784 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==14105==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==14105==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0e2020 is 176 bytes inside a block of size 768 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6A369A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E22F4: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5972BF: PySymtable_Build (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599EB1: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==14105==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==14105==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0e3020 is 16 bytes inside a block of size 40 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C5E4B9C: PyCode_Optimize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E625E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FD5: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C3262F9: add_miscstuff (clawsmailmodule.c:898)
==14105==    by 0x1C3264E7: initclawsmail (clawsmailmodule.c:936)
==14105==    by 0x1C32B104: plugin_init (python_plugin.c:669)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599FE7: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0ee020 is 34,480 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAED1: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0ee020 is 34,480 bytes inside a block of size 36,032 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C680527: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1F27: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B137: plugin_init (python_plugin.c:675)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C696035: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EB6: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==14105==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==14105==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695F79: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==14105==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0fc020 is 160 bytes inside a block of size 320 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C695EEA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69607D: PyNode_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5F1ED7: PyParser_ASTFromString (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE5C: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==14105==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C652DC5: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C599C4C: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==14105==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0fe020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C49D1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C598ACD: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59A114: PyAST_Compile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DAE76: PyRun_StringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5DBA6A: PyRun_SimpleStringFlags (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32BA85: parasite_python_init (python-hooks.c:156)
==14105==    by 0x1C32B160: plugin_init (python_plugin.c:681)
==14105==    by 0x6623FB: plugin_load (plugin.c:514)
==14105==    by 0x662A45: plugin_load_all (plugin.c:618)
==14105==    by 0x4D907D: main (main.c:1418)
==14105==  Address 0x1d0fe020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C69540B: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C63F8: _PyObject_GC_Resize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59220B: PyFrame_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583DC9: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C69541F: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C63F8: _PyObject_GC_Resize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59220B: PyFrame_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583DC9: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C695402: PyObject_Realloc (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C63F8: _PyObject_GC_Resize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C59220B: PyFrame_New (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583DC9: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C583E1A: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58282D: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1d18e020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C681838: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C681838: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C654B06: PyEval_CallObjectWithKeywords (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C57F5C2: PyEval_EvalFrameEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584C37: PyEval_EvalCodeEx (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C584D71: PyEval_EvalCode (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1d113020 is 3,632 bytes inside a block of size 5,410 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==14105==  Address 0x1d113020 is 3,632 bytes inside a block of size 5,410 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C67B36A: PyMarshal_ReadLastObjectFromFile (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680A0D: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C680B8E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5E06C0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6812EB: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C638612: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6817FC: PyImport_ImportModuleLevel (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C5D703E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6541FD: PyObject_Call (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C655F4A: PyObject_CallFunction (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65A7D9: PyImport_Import (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65BF09: PyImport_ImportModule (in /usr/lib/libpython2.7.so.1.0)
==14105== 
==14161== 
==14161== HEAP SUMMARY:
==14161==     in use at exit: 13,046,906 bytes in 71,260 blocks
==14161==   total heap usage: 246,905 allocs, 175,645 frees, 34,025,273 bytes allocated
==14161== 
==14161== LEAK SUMMARY:
==14161==    definitely lost: 41,363 bytes in 112 blocks
==14161==    indirectly lost: 6,944 bytes in 218 blocks
==14161==      possibly lost: 3,317,077 bytes in 20,484 blocks
==14161==    still reachable: 9,681,522 bytes in 50,446 blocks
==14161==         suppressed: 0 bytes in 0 blocks
==14161== Rerun with --leak-check=full to see details of leaked memory
==14161== 
==14161== For counts of detected and suppressed errors, rerun with: -v
==14161== Use --track-origins=yes to see where uninitialised values come from
==14161== ERROR SUMMARY: 1814 errors from 83 contexts (suppressed: 2 from 2)
==14166== 
==14166== HEAP SUMMARY:
==14166==     in use at exit: 13,046,976 bytes in 71,263 blocks
==14166==   total heap usage: 246,908 allocs, 175,645 frees, 34,025,343 bytes allocated
==14166== 
==14167== 
==14167== HEAP SUMMARY:
==14167==     in use at exit: 13,046,976 bytes in 71,263 blocks
==14167==   total heap usage: 246,908 allocs, 175,645 frees, 34,025,343 bytes allocated
==14167== 
==14166== LEAK SUMMARY:
==14166==    definitely lost: 41,363 bytes in 112 blocks
==14166==    indirectly lost: 6,944 bytes in 218 blocks
==14166==      possibly lost: 3,317,077 bytes in 20,484 blocks
==14166==    still reachable: 9,681,592 bytes in 50,449 blocks
==14166==         suppressed: 0 bytes in 0 blocks
==14166== Rerun with --leak-check=full to see details of leaked memory
==14166== 
==14166== For counts of detected and suppressed errors, rerun with: -v
==14166== Use --track-origins=yes to see where uninitialised values come from
==14166== ERROR SUMMARY: 1814 errors from 83 contexts (suppressed: 2 from 2)
==14167== LEAK SUMMARY:
==14167==    definitely lost: 41,363 bytes in 112 blocks
==14167==    indirectly lost: 6,944 bytes in 218 blocks
==14167==      possibly lost: 3,317,077 bytes in 20,484 blocks
==14167==    still reachable: 9,681,592 bytes in 50,449 blocks
==14167==         suppressed: 0 bytes in 0 blocks
==14167== Rerun with --leak-check=full to see details of leaked memory
==14167== 
==14167== For counts of detected and suppressed errors, rerun with: -v
==14167== Use --track-origins=yes to see where uninitialised values come from
==14167== ERROR SUMMARY: 1814 errors from 83 contexts (suppressed: 2 from 2)
==14168== 
==14169== 
==14168== HEAP SUMMARY:
==14168==     in use at exit: 13,047,041 bytes in 71,265 blocks
==14168==   total heap usage: 246,910 allocs, 175,645 frees, 34,025,408 bytes allocated
==14168== 
==14169== HEAP SUMMARY:
==14169==     in use at exit: 13,047,041 bytes in 71,265 blocks
==14169==   total heap usage: 246,910 allocs, 175,645 frees, 34,025,408 bytes allocated
==14169== 
==14169== LEAK SUMMARY:
==14168==    definitely lost: 41,363 bytes in 112 blocks
==14169==    indirectly lost: 6,944 bytes in 218 blocks
==14169==      possibly lost: 3,317,077 bytes in 20,484 blocks
==14168==      possibly lost: 3,317,077 bytes in 20,484 blocks
==14169==         suppressed: 0 bytes in 0 blocks
==14169== Rerun with --leak-check=full to see details of leaked==14168===14169== 
==14168== Rerun with --leak-check=full to see details of leaked memory
==14168== 
==14169== For counts of detected and suppressed errors, rerun with: -v
==14169== Use --track-origins=yes to see where uninitialised values come from
==14169== ERROR SUMMARY: 1814 errors from 83 contexts (suppressed: 2 from 2)
==14168== Use --track-origins=yes to see where uninitialised values come from
==14168== ERROR SUMMARY: 1814 errors from 83 contexts (suppressed: 2 from 2)
==14170== 
==14170== HEAP SUMMARY:
==14170==     in use at exit: 13,047,041 bytes in 71,265 blocks
==14170==   total heap usage: 246,910 allocs, 175,645 frees, 34,025,408 bytes allocated
==14170== 
==14171== 
==14171== HEAP SUMMARY:
==14171==     in use at exit: 13,047,041 bytes in 71,265 blocks
==14171==   total heap usage: 246,910 allocs, 175,645 frees, 34,025,408 bytes allocated
==14171== 
==14170== LEAK SUMMARY:
==14170==    definitely lost: 41,363 bytes in 112 blocks
==14170==    indirectly lost: 6,944 bytes in 218 blocks
==14170==      possibly lost: 3,317,077 bytes in 20,484 blocks
==14170==    still reachable: 9,681,657 bytes in 50,451 blocks
==14170==         suppressed: 0 bytes in 0 blocks
==14170== Rerun with --leak-check=full to see details of leaked memory
==14170== 
==14170== For counts of detected and suppressed errors, rerun with: -v
==14170== Use --track-origins=yes to see where uninitialised values come from
==14170== ERROR SUMMARY: 1814 errors from 83 contexts (suppressed: 2 from 2)
==14171== LEAK SUMMARY:
==14171==    definitely lost: 41,363 bytes in 112 blocks
==14171==    indirectly lost: 6,944 bytes in 218 blocks
==14171==      possibly lost: 3,317,077 bytes in 20,484 blocks
==14171==    still reachable: 9,681,657 bytes in 50,451 blocks
==14171==         suppressed: 0 bytes in 0 blocks
==14171== Rerun with --leak-check=full to see details of leaked memory
==14171== 
==14171== For counts of detected and suppressed errors, rerun with: -v
==14171== Use --track-origins=yes to see where uninitialised values come from
==14171== ERROR SUMMARY: 1814 errors from 83 contexts (suppressed: 2 from 2)
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0xB43C9F0: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==14105==    by 0xB422076: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==14105==    by 0xB3D5810: pixman_image_composite32 (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==14105==    by 0x6513C6D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6550CA6: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x654362A: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6543F6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6544A6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x65086ED: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6518932: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6547C4D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x650FE24: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0xB43C6E8: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==14105==    by 0xB422076: ??? (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==14105==    by 0xB3D5810: pixman_image_composite32 (in /usr/lib/x86_64-linux-gnu/libpixman-1.so.0.30.2)
==14105==    by 0x6513C6D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6550CA6: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x654362A: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6543F6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6544A6E: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x65086ED: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6518932: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x6547C4D: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105==    by 0x650FE24: ??? (in /usr/lib/x86_64-linux-gnu/libcairo.so.2.11200.2)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x336650A7: ??? (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.36.3)
==14105==    by 0x33665D48: rsvg_handle_get_pixbuf_sub (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.36.3)
==14105==    by 0x3340DEAA: ??? (in /usr/lib/x86_64-linux-gnu/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so)
==14105==    by 0x62D272B: gdk_pixbuf_loader_close (in /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.2600.4)
==14105==    by 0x62CEA7A: ??? (in /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.2600.4)
==14105==    by 0x62D02B0: gdk_pixbuf_new_from_stream_at_scale (in /usr/lib/x86_64-linux-gnu/libgdk_pixbuf-2.0.so.0.2600.4)
==14105==    by 0x557F5AD: ??? (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==14105==    by 0x5581FB9: gtk_icon_info_load_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==14105==    by 0x5582166: gtk_icon_theme_load_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==14105==    by 0x557D650: gtk_icon_set_render_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==14105==    by 0x56D04B2: gtk_widget_render_icon (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==14105==    by 0x559165B: ??? (in /usr/lib/x86_64-linux-gnu/libgtk-x11-2.0.so.0.2400.13)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x89DBDB0: re_compile_fastmap_iter.isra.30 (regcomp.c:329)
==14105==    by 0x89E8669: re_compile_fastmap (regcomp.c:281)
==14105==    by 0x89E8E8F: regcomp (regcomp.c:517)
==14105==    by 0x67C2F8: subject_get_prefix_length (utils.c:3805)
==14105==    by 0x59CB84: subject_hashtable_insert (procmsg.c:141)
==14105==    by 0x59CF2D: procmsg_get_thread_tree (procmsg.c:242)
==14105==    by 0x5BFFE2: summary_set_ctree_from_list (summaryview.c:3141)
==14105==    by 0x5BA939: summary_show (summaryview.c:1463)
==14105==    by 0x4B6B7E: folderview_selected (folderview.c:2145)
==14105==    by 0x6FD98A: claws_marshal_VOID__POINTER_INT (claws-marshal.c:348)
==14105==    by 0x7E0C13F: g_closure_invoke (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==14105==    by 0x7E1D54F: ??? (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x89DBDB0: re_compile_fastmap_iter.isra.30 (regcomp.c:329)
==14105==    by 0x89E8689: re_compile_fastmap (regcomp.c:283)
==14105==    by 0x89E8E8F: regcomp (regcomp.c:517)
==14105==    by 0x67C2F8: subject_get_prefix_length (utils.c:3805)
==14105==    by 0x59CB84: subject_hashtable_insert (procmsg.c:141)
==14105==    by 0x59CF2D: procmsg_get_thread_tree (procmsg.c:242)
==14105==    by 0x5BFFE2: summary_set_ctree_from_list (summaryview.c:3141)
==14105==    by 0x5BA939: summary_show (summaryview.c:1463)
==14105==    by 0x4B6B7E: folderview_selected (folderview.c:2145)
==14105==    by 0x6FD98A: claws_marshal_VOID__POINTER_INT (claws-marshal.c:348)
==14105==    by 0x7E0C13F: g_closure_invoke (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==14105==    by 0x7E1D54F: ??? (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==  Address 0x1cdf7020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E64A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==  Address 0x1cdf7020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==  Address 0x1ceca020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==  Address 0x1cdfa020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E8AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105==  Address 0x1cdfa020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105==  Address 0x1ceb0020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37B1: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E65A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605D15: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==  Address 0x1cc4d020 is 2,048 bytes inside a block of size 2,056 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x7E2C7F4: g_type_free_instance (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==14105==    by 0x6A9265: row_delete (gtkcmctree.c:2637)
==14105==    by 0x6A8C01: tree_delete_row (gtkcmctree.c:2487)
==14105==    by 0x6AD172: gtk_cmctree_post_recursive (gtkcmctree.c:3335)
==14105==    by 0x6ACDB7: real_clear (gtkcmctree.c:3295)
==14105==    by 0x6DF298: gtk_sctree_clear (gtksctree.c:771)
==14105==    by 0x6C5B85: gtk_cmclist_clear (gtkcmclist.c:2825)
==14105==    by 0x5BB3A1: summary_clear_list (summaryview.c:1634)
==14105==    by 0x4DB553: app_will_exit (main.c:2121)
==14105==    by 0x4ED378: app_exit_cb (mainwindow.c:4021)
==14105==    by 0x7E0C13F: g_closure_invoke (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.3400.1)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5B0: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105==  Address 0x1cc48020 is 0 bytes inside a block of size 16 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x684249: xml_free_tag (xml.c:654)
==14105==    by 0x6840DE: xml_free_node (xml.c:621)
==14105==    by 0x684126: xml_free_func (xml.c:630)
==14105==    by 0x809A854: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x809A808: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x809A808: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x809A808: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x809A808: ??? (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x809B550: g_node_traverse (in /lib/x86_64-linux-gnu/libglib-2.0.so.0.3400.1)
==14105==    by 0x68419F: xml_free_tree (xml.c:638)
==14105==    by 0x49E625: folder_write_list (folder.c:906)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E5DA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==  Address 0x1d1b6020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E63A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E89A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C60603D: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==  Address 0x1d1b6020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== 
==14105== More than 100 errors detected.  Subsequent errors
==14105== will still be recorded, but in less detail than before.
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A36BC: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E2AA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C606211: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==  Address 0x1cdc8020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A36BC: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C606412: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C606211: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6147D9: _PyImport_Fini (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EFA4: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105== 
==14105== Use of uninitialised value of size 8
==14105==    at 0x1C6BCF53: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A37CA: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6A367E: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6147D9: _PyImport_Fini (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EFA4: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69613B: PyDict_Fini (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58F009: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105==  Address 0x1d2b6020 is 1,536 bytes inside a block of size 3,072 free'd
==14105==    at 0x4C2A739: free (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==14105==    by 0x1C6A369A: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C65E882: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C613316: ??? (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C6C2133: PyDict_SetItem (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C605DAF: _PyModule_Clear (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C606211: PyImport_Cleanup (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58EF9A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C69613B: PyDict_Fini (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58F009: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105== 
==14105== Invalid read of size 4
==14105==    at 0x1C6BCF32: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695C95: PyGrammar_RemoveAccelerators (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58F01A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105==  Address 0x1ceea020 is not stack'd, malloc'd or (recently) free'd
==14105== 
==14105== Conditional jump or move depends on uninitialised value(s)
==14105==    at 0x1C6BCF3B: PyObject_Free (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C695C95: PyGrammar_RemoveAccelerators (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C58F01A: Py_Finalize (in /usr/lib/libpython2.7.so.1.0)
==14105==    by 0x1C32B277: plugin_done (python_plugin.c:717)
==14105==    by 0x662665: plugin_unload (plugin.c:545)
==14105==    by 0x662B4F: plugin_unload_all (plugin.c:639)
==14105==    by 0x4D987B: exit_claws (main.c:1668)
==14105==    by 0x4D95FA: main (main.c:1574)
==14105== 
==14105== 
==14105== HEAP SUMMARY:
==14105==     in use at exit: 28,421,190 bytes in 83,097 blocks
==14105==   total heap usage: 2,288,198 allocs, 2,205,101 frees, 210,685,308 bytes allocated
==14105== 
==14105== LEAK SUMMARY:
==14105==    definitely lost: 48,361 bytes in 134 blocks
==14105==    indirectly lost: 39,904 bytes in 1,243 blocks
==14105==      possibly lost: 20,524,016 bytes in 52,169 blocks
==14105==    still reachable: 7,808,909 bytes in 29,551 blocks
==14105==         suppressed: 0 bytes in 0 blocks
==14105== Rerun with --leak-check=full to see details of leaked memory
==14105== 
==14105== For counts of detected and suppressed errors, rerun with: -v
==14105== Use --track-origins=yes to see where uninitialised values come from
==14105== ERROR SUMMARY: 2494 errors from 108 contexts (suppressed: 2 from 2)
